code/index.html

```diff
@@ -3,28 +3,226 @@
+        </div>
       </div>
+      <div class="event-container">
+        <img class="event-image" src= "images/fourth-mobile-image.jpg" alt="table setting">
+        <div class="img-wrap">
+          <img class="event-image-desktop" src="images/second-desktop-image.jpg" alt="table setting outdoors">
+          <div class="event-header">Events</div>
+          <div class="event-text">We have beautiful vintage furnitures, porcelain and decoration for hire </div>
+        </div>
+      </div>
+    </section>
+      
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+        <!-- get to know -->
+      <h2 class="info">Get to know us</h2>
+      <section class="section-container">
+        <div class="portfolio-container">
+          <div class="img-wrap">
+            <img class="portfolio-image" src= "images/second-image-mobile.jpg" alt="guitar decorated with flowers on chair">
+            <div class="portfolio-header">Portfolio</div>              
+            <div class="portfolio-text">Browse our previous event decorations</div>
+          </div>
+        </div>
+        <div class="about-container">
+          <div class="img-wrap">
+          <img class="about-image" src= "images/about-us.jpg" alt="2 people running on a field">
+          <div class="about-header">Our story</div>
+        </div>
+       </div>
+      </section>
+    
+    
+    <!-- Accordion -->
+   <section-container>
+     <div class="accordion-background">
+      <img class="accordion-image" src="images/fifth-mobile-image.jpg" alt="cozy picnic in the evening">
+      
+      <div class="accordion-container">
+        <h1 class="accordion-header">FAQ</h1>
+        
+        <button id="Question1" class="accordion1">Where are you located?</button>
+        <div class="panel">
+          <p>lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut abore et dolore magna aliqua.</p>
+        </div>
+          
+        <button id="Question2" class="accordion">What services do you offer?</button>
+        <div class="panel">
+          <p>lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut abore et dolore magna aliqua.</p>
+        </div>
+          
+        <button id="Question3" class="accordion3">Where are you located?</button>
+        <div class="panel">
+           <p>lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut abore et dolore magna aliqua.</p>
+        </div>
+          
+        <button id="Question4" class="accordion">What services do you offer?</button>
+        <div class="panel">
+          <p>lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut abore et dolore magna aliqua.</p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+    
+      <!-- Signup form -->
```

Comment:
Also a supersmall detail that the text on the button is not centered.
The form is really nicely made - the text fields being lines only is very neat!